scalalib/src/main/scala/mill/scalalib/GenIdea.scala

```diff
@@ -72,16 +72,21 @@ object GenIdea {
     }
 
     val moduleFiles = resolved.map{ case (path, resolvedDeps, mod) =>
-      val Seq(sourcePath: PathRef) =
-        evaluator.evaluate(OSet(mod.sources)).values
+      val Seq(sourcesPathRef: PathRef, generatedSourcePathRefs: Seq[PathRef], allSourcesPathRefs: Seq[PathRef]) =
+        evaluator.evaluate(OSet(mod.sources, mod.generatedSources, mod.allSources)).values
+
+      val generatedSourcePaths = generatedSourcePathRefs.map(_.path)
+      val normalSourcePaths = (allSourcesPathRefs.map(_.path).toSet -- generatedSourcePaths.toSet).toSeq
```

Comment:
Isn't `sourcesPathRef ` == `normalSourcePaths ` ?

Reply:
@rockjam 

what I thought is: if you override allSources to add addtional source code folder then `normalSourcePaths` contain other source folders.

But now I am not sure if it is designed like this, can you have two source folders in a module? Because seems all docs, sourcejar is generated only using the main source one. So maybe `allSources - sources = generatedSources`??

Reply:
I will look into if it is usually needed to have multiple non-generated source folders for a module, in the meanwhile, what's your opinion? @lihaoyi